lib/src/main/scala/spinal/lib/CrossClock.scala

```diff
@@ -64,12 +65,16 @@ class BufferCC[T <: Data](val dataType: T, init :  => T, val bufferDepth: Option
 
   buffers(0) := io.dataIn
   buffers(0).addTag(crossClockDomain)
+  inputAttributes.foreach(buffers(0).addTag)
   for (i <- 1 until finalBufferDepth) {
     buffers(i) := buffers(i - 1)
     buffers(i).addTag(crossClockBuffer)
```

Comment:
also the case for my original code, but I think we are never putting any info for correctly setting the `set_property ASYNC_REG TRUE` property on the buffers in the receiving CD so that P&R will keep the path between them short?

Reply:
I think this is addressed by `crossClockBuffer` and later realized into `async_reg` by `PhaseDevice`: https://github.com/SpinalHDL/SpinalHDL/blob/83efd666ab5aa1d513d3f55ca30a281b7bb8f8cf/core/src/main/scala/spinal/core/internals/Phase.scala#L1368-L1372

Reply:
Result: 
```verilog
module BufferCC_2 (
  input  wire          io_dataIn,
  output wire          io_dataOut,
  input  wire          clk,  
  input  wire          cmacTxClock_reset_synchronized
);

  (* async_reg = "true" *) reg                 buffers_0;
  (* async_reg = "true" *) reg                 buffers_1;

  assign io_dataOut = buffers_1;
  always @(posedge clk) begin
    if(cmacTxClock_reset_synchronized) begin
      buffers_0 <= 1'b0;                                                                                                                                                                     
      buffers_1 <= 1'b0; 
    end else begin 
      buffers_0 <= io_dataIn;
      buffers_1 <= buffers_0;
    end   
  end


endmodule
```